src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -222,6 +222,18 @@
     @Parameter
     private String[] ignoredUnusedDeclaredDependencies = new String[0];
 
+    /**
+     * List of project packaging that will be ignored.
+     *
+     * <p>
+     * by default <b>pom</b> and <b>ear</b> are ignored.
+     * </p>
+     *
+     * @since 3.2.1
+     */
+    @Parameter
+    private List<String> ignoredPackagings = Arrays.asList( "pom", "ear" );
```

Comment:
I'd also like to see `rar` here.

Reply:
- dock will be fixed.
- Maven 3.2.5 is used on ASF jenkins ... so we need workaround ...
- I prefer add new package with new issue and new PR - I don't want to change scope of this

Reply:
>     * Maven 3.2.5 is used on ASF jenkins ... so we need workaround ...

:disappointed: 

>     * I prefer add new package with new issue and new PR - I don't want to change scope of this
:+1:

Reply:
Now look like standard defaults

![image](https://user-images.githubusercontent.com/3578921/141685860-6bbe5ea2-f25b-4b12-9474-048ead91a6d8.png)